src/lib/aws/aws_common_resources.py

```diff
@@ -0,0 +1,6 @@
+
+
+class AWS_Common_Resources:
+    @classmethod
+    def get_Lambda_Powertools_Layer_Arn(cls, region):
```

Comment:
The method should be named in lowercase.

Reply:
fixed